CRITICAL - 2023-09-29 19:28:03 --> Undefined array key "fechaNacimiento"
in APPPATH\Controllers\Alumno.php on line 40.
 1 APPPATH\Controllers\Alumno.php(40): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "fechaNacimiento"', 'C:\\xampp\\htdocs\\app_web\\app\\Controllers\\Alumno.php', 40)
 2 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\Alumno->insert()
 3 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Alumno))
 4 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\app_web\\public\\index.php')
CRITICAL - 2023-09-29 19:28:58 --> Undefined array key "fechaNacimiento"
in APPPATH\Controllers\Alumno.php on line 40.
 1 APPPATH\Controllers\Alumno.php(40): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "fechaNacimiento"', 'C:\\xampp\\htdocs\\app_web\\app\\Controllers\\Alumno.php', 40)
 2 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\Alumno->insert()
 3 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Alumno))
 4 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\app_web\\public\\index.php')
ERROR - 2023-09-29 19:30:41 --> Error connecting to the database: mysqli_sql_exception: Access denied for user ''@'localhost' (using password: NO) in C:\xampp\htdocs\app_web\system\Database\MySQLi\Connection.php:184
Stack trace:
#0 C:\xampp\htdocs\app_web\system\Database\MySQLi\Connection.php(184): mysqli->real_connect('127.0.0.1', '', Object(SensitiveParameterValue), 'estudiantec', 3306, '', 0)
#1 C:\xampp\htdocs\app_web\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\app_web\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\app_web\system\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `Al...', Array, false)
#4 C:\xampp\htdocs\app_web\system\Model.php(319): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\app_web\system\BaseModel.php(788): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\app_web\system\Model.php(715): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\app_web\app\Controllers\Alumno.php(42): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\app_web\system\CodeIgniter.php(919): App\Controllers\Alumno->insert()
#9 C:\xampp\htdocs\app_web\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Alumno))
#10 C:\xampp\htdocs\app_web\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\app_web\public\index.php(79): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\app_web\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#13 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user ''@'localhost' (using password: NO) in C:\xampp\htdocs\app_web\system\Database\MySQLi\Connection.php:229
Stack trace:
#0 C:\xampp\htdocs\app_web\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\app_web\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\app_web\system\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `Al...', Array, false)
#3 C:\xampp\htdocs\app_web\system\Model.php(319): CodeIgniter\Database\BaseBuilder->insert()
#4 C:\xampp\htdocs\app_web\system\BaseModel.php(788): CodeIgniter\Model->doInsert(Array)
#5 C:\xampp\htdocs\app_web\system\Model.php(715): CodeIgniter\BaseModel->insert(Array, true)
#6 C:\xampp\htdocs\app_web\app\Controllers\Alumno.php(42): CodeIgniter\Model->insert(Array)
#7 C:\xampp\htdocs\app_web\system\CodeIgniter.php(919): App\Controllers\Alumno->insert()
#8 C:\xampp\htdocs\app_web\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Alumno))
#9 C:\xampp\htdocs\app_web\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\app_web\public\index.php(79): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\app_web\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#12 {main}
CRITICAL - 2023-09-29 19:30:41 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user ''@'localhost' (using password: NO)
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `Alumno` (`nombre`, `sexo`, `fechaNacimiento`) VALUES (:nombre:, :sexo:, :fechaNacimiento:)', [...], false)
 3 SYSTEMPATH\Model.php(319): CodeIgniter\Database\BaseBuilder->insert()
 4 SYSTEMPATH\BaseModel.php(788): CodeIgniter\Model->doInsert([...])
 5 SYSTEMPATH\Model.php(715): CodeIgniter\BaseModel->insert([...], true)
 6 APPPATH\Controllers\Alumno.php(42): CodeIgniter\Model->insert([...])
 7 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\Alumno->insert()
 8 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Alumno))
 9 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\app_web\\public\\index.php')
CRITICAL - 2023-09-29 19:33:10 --> Undefined variable $alumnoModel
in APPPATH\Controllers\Alumno.php on line 17.
 1 APPPATH\Controllers\Alumno.php(17): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $alumnoModel', 'C:\\xampp\\htdocs\\app_web\\app\\Controllers\\Alumno.php', 17)
 2 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\Alumno->mostrar()
 3 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Alumno))
 4 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\app_web\\public\\index.php')
CRITICAL - 2023-09-29 19:34:23 --> Undefined variable $alumnoModel
in APPPATH\Controllers\Alumno.php on line 17.
 1 APPPATH\Controllers\Alumno.php(17): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $alumnoModel', 'C:\\xampp\\htdocs\\app_web\\app\\Controllers\\Alumno.php', 17)
 2 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\Alumno->mostrar()
 3 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Alumno))
 4 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\app_web\\public\\index.php')
CRITICAL - 2023-09-29 19:34:26 --> Undefined variable $alumnoModel
in APPPATH\Controllers\Alumno.php on line 17.
 1 APPPATH\Controllers\Alumno.php(17): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $alumnoModel', 'C:\\xampp\\htdocs\\app_web\\app\\Controllers\\Alumno.php', 17)
 2 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\Alumno->mostrar()
 3 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Alumno))
 4 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\app_web\\public\\index.php')
CRITICAL - 2023-09-29 19:34:40 --> Undefined variable $alumnoModel
in APPPATH\Controllers\Alumno.php on line 17.
 1 APPPATH\Controllers\Alumno.php(17): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $alumnoModel', 'C:\\xampp\\htdocs\\app_web\\app\\Controllers\\Alumno.php', 17)
 2 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\Alumno->mostrar()
 3 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Alumno))
 4 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\app_web\\public\\index.php')
CRITICAL - 2023-09-29 19:35:04 --> Undefined variable $alumnoModel
in APPPATH\Controllers\Alumno.php on line 17.
 1 APPPATH\Controllers\Alumno.php(17): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $alumnoModel', 'C:\\xampp\\htdocs\\app_web\\app\\Controllers\\Alumno.php', 17)
 2 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\Alumno->mostrar()
 3 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Alumno))
 4 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\app_web\\public\\index.php')
CRITICAL - 2023-09-29 19:37:17 --> Undefined variable $alumnoModel
in APPPATH\Controllers\Alumno.php on line 17.
 1 APPPATH\Controllers\Alumno.php(17): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $alumnoModel', 'C:\\xampp\\htdocs\\app_web\\app\\Controllers\\Alumno.php', 17)
 2 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\Alumno->mostrar()
 3 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Alumno))
 4 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\app_web\\public\\index.php')
CRITICAL - 2023-09-29 19:37:55 --> Undefined variable $alumnoModel
in APPPATH\Controllers\Alumno.php on line 17.
 1 APPPATH\Controllers\Alumno.php(17): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $alumnoModel', 'C:\\xampp\\htdocs\\app_web\\app\\Controllers\\Alumno.php', 17)
 2 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\Alumno->mostrar()
 3 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Alumno))
 4 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\app_web\\public\\index.php')
CRITICAL - 2023-09-29 19:38:10 --> Undefined variable $alumnoModel
in APPPATH\Controllers\Alumno.php on line 17.
 1 APPPATH\Controllers\Alumno.php(17): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $alumnoModel', 'C:\\xampp\\htdocs\\app_web\\app\\Controllers\\Alumno.php', 17)
 2 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\Alumno->mostrar()
 3 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Alumno))
 4 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\app_web\\public\\index.php')
ERROR - 2023-09-29 19:38:39 --> Error connecting to the database: mysqli_sql_exception: Access denied for user ''@'localhost' (using password: NO) in C:\xampp\htdocs\app_web\system\Database\MySQLi\Connection.php:184
Stack trace:
#0 C:\xampp\htdocs\app_web\system\Database\MySQLi\Connection.php(184): mysqli->real_connect('127.0.0.1', '', Object(SensitiveParameterValue), 'estudiantec', 3306, '', 0)
#1 C:\xampp\htdocs\app_web\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\app_web\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\app_web\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\app_web\system\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\app_web\system\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\app_web\app\Controllers\Alumno.php(17): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\app_web\system\CodeIgniter.php(919): App\Controllers\Alumno->mostrar()
#8 C:\xampp\htdocs\app_web\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Alumno))
#9 C:\xampp\htdocs\app_web\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\app_web\public\index.php(79): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\app_web\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#12 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user ''@'localhost' (using password: NO) in C:\xampp\htdocs\app_web\system\Database\MySQLi\Connection.php:229
Stack trace:
#0 C:\xampp\htdocs\app_web\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\app_web\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\app_web\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\app_web\system\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\app_web\system\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
#5 C:\xampp\htdocs\app_web\app\Controllers\Alumno.php(17): CodeIgniter\BaseModel->findAll()
#6 C:\xampp\htdocs\app_web\system\CodeIgniter.php(919): App\Controllers\Alumno->mostrar()
#7 C:\xampp\htdocs\app_web\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Alumno))
#8 C:\xampp\htdocs\app_web\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\app_web\public\index.php(79): CodeIgniter\CodeIgniter->run()
#10 C:\xampp\htdocs\app_web\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#11 {main}
CRITICAL - 2023-09-29 19:38:39 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user ''@'localhost' (using password: NO)
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `Alumno`
WHERE `Alumno`.`deleted_at` IS NULL', [], false)
 3 SYSTEMPATH\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Controllers\Alumno.php(17): CodeIgniter\BaseModel->findAll()
 6 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\Alumno->mostrar()
 7 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Alumno))
 8 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\app_web\\public\\index.php')
